<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topviewplotdlg</class>
 <widget class="QDialog" name="topviewplotdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qwt plots list</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="btnframe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbClosePlots">
        <property name="toolTip">
         <string>Close all plots</string>
        </property>
        <property name="text">
         <string>CLOSE</string>
        </property>
        <property name="icon">
         <iconset resource="plib1.qrc">
          <normaloff>:/icons/img/cansel1.png</normaloff>:/icons/img/cansel1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbShowEverything">
        <property name="toolTip">
         <string>show all plots</string>
        </property>
        <property name="text">
         <string>show and align all</string>
        </property>
        <property name="icon">
         <iconset resource="plib1.qrc">
          <normaloff>:/icons/img/binokl.png</normaloff>:/icons/img/binokl.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbShowSimple">
        <property name="toolTip">
         <string>show all plots</string>
        </property>
        <property name="text">
         <string>Show all plots</string>
        </property>
        <property name="icon">
         <iconset resource="plib1.qrc">
          <normaloff>:/icons/img/thunderstorm-icon.png</normaloff>:/icons/img/thunderstorm-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>103</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="viewframe">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTreeView" name="tv"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="plib1.qrc"/>
 </resources>
 <connections/>
</ui>
